### PR TITLE
Fix marking branch for removal after merge in Gitlab

### PR DESCRIPTION
**Pull request by WouldYouKindly**
Hi.

I renamed `should_remove_source_branch` to `remove_source_branch`, and flag is now correctly set on merge request.

Tested on GitLab Community Edition 11.9.4 

According to https://gitlab.com/gitlab-org/gitlab-ce/blob/master/doc/api/merge_requests.md#L745

### CODE REVIEW

File: pyup/providers/gitlab.py [hunk 1]
```diff
@@ -184,7 +184,7 @@ def create_pull_request(self, repo, title, body, base_branch, new_branch, pr_lab
                 'title': title,
                 'description': body,
                 'pr_label': pr_label,
-                'should_remove_source_branch': config.gitlab.should_remove_source_branch
+                'remove_source_branch': config.gitlab.remove_source_branch
```
Comment (htgoebel) [inline pyup/providers/gitlab.py]: I suggest to reduce this change to only this line - which is the actual fix.

Comment (WouldYouKindly) [reply pyup/providers/gitlab.py]: Hi. Are you `pyup` commiter? The repository seems kind of dead.

Comment (htgoebel) [reply pyup/providers/gitlab.py]: @WouldYouKindly: Nope, I'm just a user

Comment (rafaelpivato) [reply pyup/providers/gitlab.py]: That sounds like the best option. And sorry for the late response, @WouldYouKindly

**Comment by rafaelpivato** [comment]
Thanks for showing where the problem was, @WouldYouKindly :+1:

**Comment by WouldYouKindly** [comment]
@rafaelpivato thanks for taking the time to fix it!

**Pull request merged**